src/sentry/static/sentry/app/views/organizationIntegrations/constants.tsx

```diff
@@ -52,3 +52,24 @@ export const POPULARITY_WEIGHT: {
   'amazon-sqs': 2,
   splunk: 2,
 } as const;
+
+export const documentIntegrations = {
```

Comment:
@NisanthanNanthakumar You should create a type for document-based integrations. This will prevent the human error cases you were concerned about because if you are missing a field like `author`, it won't compile.

Reply:
@NisanthanNanthakumar Nvm, you have a type but you aren't using it here. I think this is what you want:
```
export const documentIntegrations: {[key: string]: DocumentIntegration} = 
```

Reply:
@scefali If I do that, then we can use `as const`. I think `as const` is more useful, b/c it makes it all readonly

Reply:
I'm less concerned about the read-only aspects and more concerned about typos in the JSON not matching the type. The latter types of bugs are far more common than people accidentally mutating a constant which should be caught in a PR review.

Reply:
@scefali I understand. Updated.